faq/dump-css.md

```diff
@@ -0,0 +1,10 @@
+
+
+
+## Dumping CSS protected discs with Aaru
+
+Now that you know the basics, you're ready to start the dumping process! The first step is to have a compatible DVD/Blu-ray drive installed and ready for Aaru to use. This includes making sure it's authenticated for the region you intend to dump, as Aaru does NOT bypass the region authentication. After that, you have to ensure that Aaru is configured to dump CSS protected discs. This is asked on the first run of Aaru, as this action is strictly illegal in certain areas. Once you have verified that it IS legal according to your local laws, you can make sure that decryption is enabled by running ```Aaru configure``` and selecting "Y" when it asks you "Do you want to enable decryption of copy protected media?". Next, you have to consider what options you want to use, the most important being ```--store-encrypted``` and ```--title-keys```. When ```--title-keys``` is enabled, Aaru will then bruteforce the title keys on a per sector basis. While other software only does this to obtain a title key once (which does produce functional decrypted images every or almost every time), Aaru does this for every sector for the sake of complete preservation. So while this is needed for Aaru in order to produce a decrypted copy of the disc, it takes a very long time. As in at least overnight long, and quite possibly a fair bit longer, especially for dual layer DVDs. However, if you want to dump the disc with ```--store-encrypted``` enabled, it will store an image with the CSS encryption intact, which means that ```--title-keys``` can safely be disabled. An encrypted image can be easily decrypted with other software (assuming it's converted to a commonly supported format such as ISO), so if you want to make a quick dump that's not 100% geared towards preservation, this is the way to go. With the title keys, it is possible to decrypt and encrypt the disc image at will, but Aaru does not current have this functionality. It is planned to be added in a future update.
```

Comment:
do you know about paragraphs?

Reply:
Aaru does NOT BRUTEFORCE, it negotiates and requests the drive, that's why it's so slow, bruteforce would be faster :rofl: